src/goPath.ts

```diff
@@ -14,6 +14,7 @@ import { showGoStatus, hideGoStatus } from './goStatus'
 var binPathCache : { [bin: string]: string;} = {}
```

Comment:
Indeed - looks like we've never been writing to the cache.  We should either do that or remove the cache code for now pending figuring out how much of a perf hit this codepath is.